source/frontend/src/features/contacts/hooks/useContactDetail.ts

```diff
@@ -6,24 +6,37 @@ import { toast } from 'react-toastify';
 
 import { useApiContacts } from '@/hooks/pims-api/useApiContacts';
 import { IApiError } from '@/interfaces/IApiError';
-import { IContact } from '@/interfaces/IContact';
+import { ApiGen_Concepts_Organization } from '@/models/api/generated/ApiGen_Concepts_Organization';
+import { ApiGen_Concepts_Person } from '@/models/api/generated/ApiGen_Concepts_Person';
 import { logError } from '@/store/slices/network/networkSlice';
 
+export interface IContact {
+  person: ApiGen_Concepts_Person;
+  organization: ApiGen_Concepts_Organization;
```

Comment:
nit: I'd like this type to include null - ie `ConceptPerson | null` to make clear that they can be null